marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/entities/search/EntitySearchManager.java

```diff
@@ -62,6 +63,13 @@
 
     private static final String MASTERING_AUDIT_COLLECTION_NAME = "mdm-auditing";
     private static final String[] IGNORED_SM_COLLECTION_SUFFIX = {"auditing", "notification"};
+    private static final List<String> ARTIFACT_COLLECTION_NAMES = Arrays.asList(
+            "hub-core-artifact",
```

Comment:
I don't think this should be included because it raises the question of "What's in hub-core-artifact that's not in the 4 collections listed below?"

Reply:
Should these collections be included as well ?
 http://marklogic.com/data-hub/steps/mapping|ingestion|mastering|matching|merging, 
http://marklogic.com/data-hub/mappings .

Reply:
The steps collection covers all of those, and "mappings" won't exist in an HC app. Though - they could exist - so probably better to exclude those just to be safe.

Reply:
@srinathgit artifacts in these collections  http://marklogic.com/data-hub/steps/mapping|ingestion|mastering|matching|merging and http://marklogic.com/data-hub/mappings also have http://marklogic.com/data-hub/steps collection added

Reply:
@srinathgit @rjrudin Added the collections as suggested. Also removed hub-core-artifact collection